events/pull-meetup.js

```diff
@@ -0,0 +1,100 @@
+var request = require('request').defaults({json:true, headers:{'user-agent':'pull-meeting-0.1'}})
+  , url = 'https://api.meetup.com/2/groups'
+  , auth = process.env.MEETUP_TOKEN
+  , qs = require('querystring')
+  , yml = require('./yaml-sync')
+  , opts =
+    { topic: 'nodejs'
+    , category: 34
+    , upcoming_events: true
+    , key: auth
+    }
+  , allresults = []
+  , u = url+'?'+qs.stringify(opts)
+  ;
+
+var countryMap =
+ { ES: 'Africa'
+ , MU: 'Africa'
+ , NG: 'Africa'
+ , KE: 'Africa'
+ , ZA: 'Africa'
+ , IL: 'Asia'
+ , TH: 'Asia'
+ , KR: 'Asia'
+ , RU: 'Asia'
+ , ID: 'Asia'
+ , PH: 'Asia'
+ , IN: 'Asia'
+ , HK: 'Asia'
+ , GR: 'Europe'
+ , DE: 'Europe'
+ , RO: 'Europe'
+ , MT: 'Europe'
+ , GH: 'Europe'
+ , IE: 'Europe'
+ , FI: 'Europe'
+ , SE: 'Europe'
+ , UA: 'Europe'
+ , AT: 'Europe'
+ , HU: 'Europe'
+ , CH: 'Europe'
+ , IS: 'Europe'
+ , GB: 'Europe'
+ , DK: 'Europe'
+ , EE: 'Europe'
+ , BE: 'Europe'
+ , NO: 'Europe'
+ , NL: 'Europe'
+ , FR: 'Europe'
+ , PL: 'Europe'
+ , SK: 'Europe'
+ , CA: 'North America'
+ , US: 'North America'
+ , AR: 'Latin America'
+ , PE: 'Latin America'
```

Comment:
I only have all of the ones in here that we need for the meetups currently coming out of meetup.com, I wonder why there isn't one showing up for Columbia.